fixtures/helpers.js

```diff
@@ -0,0 +1,104 @@
+import * as ReactDOM from "react-dom";
+
+export async function sleep(duration = 0) {
+    return new Promise((resolve, reject) => {
+        setTimeout(resolve, duration);
+    });
+}
+
+export async function simulate(event) {
+    await fetch("http://localhost:3000/simulate", {
+        method: "POST",
+        headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+            offsetX: window.screenX,
+            offsetY: window.screenY + (window.outerHeight - window.innerHeight),
+            event: event,
+        }),
+    });
+}
+
+let container;
+
+export async function render(element) {
+    return new Promise((resolve, reject) => {
+        ReactDOM.render(element, container, () => {
+            setTimeout(() => resolve(container), 0);
+        });
+    });
+}
+
+let currentTest = null;
+
+export async function runTests() {
+    for (const test of tests) {
+        currentTest = test;
```

Comment:
This is kind of hacky, but I couldn't figure out a better way of passing the test title to `toMatchScreenshot`.